lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -59,10 +63,14 @@ def call_pipeline(*args, &block)
               pin.tracer.trace(Datadog::Contrib::Redis::Ext::SPAN_COMMAND) do |span|
                 span.service = pin.service
                 span.span_type = Datadog::Contrib::Redis::Ext::TYPE
-                commands = args[0].commands.map { |c| Datadog::Contrib::Redis::Quantize.format_command_args(c) }
-                span.resource = commands.join("\n")
+                if datadog_configuration[:command_args]
+                  commands = args[0].commands.map { |c| Datadog::Contrib::Redis::Quantize.format_command_args(c) }
+                  span.resource = commands.join("\n")
+                  span.set_metric Datadog::Contrib::Redis::Ext::METRIC_PIPELINE_LEN, commands.length
+                else
+                  span.resource = Ext::PIPELINE_RESOURCE
```

Comment:
Is there any reason `#get_verb` wouldn't work for pipelines?
```suggestion
                  commands = args[0].commands.map { |c| Datadog::Contrib::Redis::Quantize.get_verb(c) }
```

Reply:
Yeah good call.